hugegraph-llm/src/hugegraph_llm/config/config.py

```diff
@@ -33,6 +33,16 @@
 yaml_file_path = os.path.join(package_path, f"src/hugegraph_llm/resources/demo/{F_NAME}")
 
 
+def read_dotenv() -> dict[str, Optional[str]]:
+    """Read a .env file in the given root path."""
+    env_config = dotenv_values(f"{env_path}")
+    log.info("Loading %s successfully!", env_path)
+    for key, value in env_config.items():
+        if key not in os.environ:
+            os.environ[key] = value or ""
+    return env_config
+
+
```

Comment:
what's for? (no code diff?)

Reply:
The reason is that the function is in between two classes, which I personally feel is a bit odd, so I put the function on top, so that the two classes are next to each other